docs/source/markdown/podman-systemd.unit.5.md

```diff
@@ -887,6 +914,114 @@ The (optional) name of the Podman volume. If this is not specified, the default
 `systemd-%N` is used, which is the same as the unit name but with a `systemd-` prefix to avoid
 conflicts with user-managed volumes.
 
+## Image units [Image]
+
+Image files are named with a `.image` extension and contain a section `[Image]` describing the
+container image pull command. The generated service is a one-time command that ensures that the image
+exists on the host, pulling it if needed.
+
+Using image units allows containers and volumes to depend on images being automatically pulled. This is
+particularly interesting when using special options to control image pulls.
+
+Valid options for `[Image]` are listed below:
+
+| **[Image] options**                 | **podman image pull equivalent**                |
+|-------------------------------------|-------------------------------------------------|
+| AllTags=true                        | --all-tags                                      |
+| Arch=aarch64                        | --arch=aarch64                                  |
+| AuthFile=/etc/registry/auth\.json   | --authfile=/etc/registry/auth\.json             |
+| CertDir=/etc/registery/certs        | --cert-dir=/etc/registery/certs                 |
+| Creds=myname\:mypassword            | --creds=myname\:mypassword                      |
+| DecryptionKey=/etc/registery\.key   | --decryption-key=/etc/registery\.key            |
+| Image=quay.io/centos/centos\:latest | podman image pull quay.io/centos/centos\:latest |
+| OS=windows                          | --os=windows                                    |
+| PodmanArgs=--os=linux               | --os=linux                                      |
+| Platform=linux/x86_64               | --platform=linux/x86_64                         |
+| TLSVerify=false                     | --tls-verify=false                              |
+| Variant=arm/v7                      | --variant=arm/v7                                |
+
+### `AllTags=`
+
+All tagged images in the repository are pulled.
+
+This is equivalent to the Podman `--all-tags` option.
+
+### `Arch=`
+
+Override the architecture, defaults to hosts, of the image to be pulled.
+
+This is equivalent to the Podman `--arch` option.
+
+### `AuthFile=`
+
+Path  of  the  authentication file.
```

Comment:
Here and below is see a number of double or triple whitespaces.

Reply:
Done